lib/swarm/tracker/tracker.ex

```diff
@@ -120,9 +120,13 @@ defmodule Swarm.Tracker do
     # wait for node list to populate
     nodelist = Enum.reject(Node.list(:connected), &ignore_node?/1)
 
-    strategy = Node.self
-           |> Strategy.create()
-           |> Strategy.add_nodes(nodelist)
+    strategy =
+      case ignore_node?(Node.self) do
```

Comment:
When we register the workers in the ignored nodes, we should not to put the current ignored node in its strategy(ring), or when we register the workers in the ignored nodes, the started workers would run in the ignored nodes, here are the details to reproduce https://github.com/bitwalker/swarm/issues/31, hope it's clear for you.

Thanks for your reply.